### ISSUE TITLE
New ios sdk version is available 

### ISSUE THREAD

**Issue by troublediehard** [bugfender/rn-bugfender#6]
<img width="955" alt="Screen Shot 2019-10-01 at 12 34 55 PM" src="https://user-images.githubusercontent.com/6594232/65951111-0853d200-e448-11e9-9202-42aa03276da1.png">
Are there plans to update rn lib as well?

**Comment by troublediehard** [comment]
@RubenVot any updates?

**Comment by RubenVot** [comment]
Yes, we are now working on this. We want to release the new version of iOS compatible with cocoapods 1.9 first and will release the RN later. 

Sorry for the delay.
